requirements.txt

```diff
@@ -16,3 +16,4 @@ Flask-SSLify==0.1.5
 alembic==0.8.3
 setuptools==36.0.1
 flask-secure-headers==0.6
+Flask-pyoidc==1.1.0
```

Comment:
I was trying to figure out if it is good practice to pin every requirement's dependencies. Seems there is some disagreement. As for PulseGuardian, I think some are pinned and some aren't, so the worst of both worlds, heh.